bukkit/src/main/java/dev/jsinco/brewery/bukkit/breweries/BukkitBarrel.java

```diff
@@ -184,6 +184,13 @@ private void updateInventory() {
                 brews[iFinal] = brew;
             });
         }
+        if (time % 32 == 0) {
+            getInventory().getViewers()
+                    .stream()
+                    .filter(Player.class::isInstance)
+                    .map(Player.class::cast)
+                    .forEach(Player::updateInventory);
+        }
```

Comment:
Honestly, I would just do it with a simple for(each) loop, because this method is executed quite often and a bit more performance is always welcome.
```suggestion
        if (time % 32 == 0) {
            for (HumanEntity humanEntity : inventory.getViewers()) {
                if (humanEntity instanceof Player player)
                    player.updateInventory();
            }
        }
```

Reply:
Innocent until proven guilty 🤫

Reply:
But sure